packages/zsh-completions/build.sh

```diff
@@ -0,0 +1,15 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/zsh-users/zsh-completions
+TERMUX_PKG_DESCRIPTION="Additional completion definitions for Zsh"
+TERMUX_PKG_LICENSE=BSD
```

Comment:
The license is Zsh license, not `BSD`. Specify `custom` for `TERMUX_PKG_LICENSE`.